docs/csharp/index.yml

```
@@ -0,0 +1,107 @@
+          - text: Introductory tutorials
+            url: tutorials/intro-to-csharp/index.md
+      - linkListType: video
+        links:
+          - text: "What is C#?"
+            url: https://channel9.msdn.com/Series/CSharp-101/What-is-C/player
+          - text: "C# 101 video series"
+            url: https://aka.ms/dotnet3-csharp
+
+  - title: "C# language fundamentals"
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: "A tour of C#"
+            url: tour-of-csharp/index.md
+          - text: "Inside a C# program"
+            url: programming-guide/inside-a-program/index.md
+          - text: Main() and command-line arguments
+            url: programming-guide/main-and-command-args/index.md
+      - linkListType: concept
+        links:
+          - text: "The C# and .NET type system"
+            url: programming-guide/types/index.md
+          - text: Language Integrated Query (LINQ)
+            url: linq/index.md
+          - text: async and await
+            url: programming-guide/concepts/async/index.md
+      - linkListType: tutorial
+        links:
+          - text: Object oriented inheritance
+            url: tutorials/inheritance.md
+          - text: Use LINQ to query data
+            url: tutorials/working-with-linq.md
+          - text: Format test using string interpolation
+            url: tutorials/string-interpolation.md
+          - text: Fundamental language tutorials
+            url: tutorials/index.md
+  
+  - title: "What's new in C#"
+    linkLists:
+      - linkListType: whats-new
+        links:
+          - text: "What's new in C# 8.0"
+            url: whats-new/csharp-8.md
+      - linkListType: overview
+        links:
+          - text: Pattern matching
+            url: pattern-matching.md
+          - text: Nullable reference types
+            url: nullable-references.md
+      - linkListType: concept
+        links:
+          - text: Annotating your APIs for nullability
+            url: nullable-attributes.md
+      - linkListType: tutorial
+        links:
+          - text: Consume and generate async streams
+            url: tutorials/generate-consume-asynchronous-stream.md
+          - text: Develop with nullable reference types
+            url: tutorials/nullable-reference-types.md
+          - text: Upgrade existing code to use nullable reference types
+            url: tutorials/upgrade-to-nullable-references.md
+          - text: Use pattern matching for more expressive algorithms
+            url: tutorials/pattern-matching.md
+          - text: Safely version interfaces with default interface members
+            url: tutorials/default-interface-methods-versions.md
+          - text: Mix in functionality with default interface members
+            url: tutorials/mixins-with-default-interface-methods.md
+          - text: Work with subsets using ranges and indices
+            url: tutorials/ranges-indexes.md
+
+  - title: "Language reference"
```

```suggestion
  - title: "C# language reference"
```

For consistency with the fundamentals list title.